benchmarks/mmlu/mmlu_eval.py

```diff
@@ -49,8 +51,8 @@
 
 import jax
 
-from mmlu_categories import categories
-from mmlu_categories import subcategories
+from .mmlu_categories import categories
+from .mmlu_categories import subcategories
```

Comment:
Nit: We generally use the absolute module path instead of the relative one. Is it possible to use that instead? I think it's this but please double check:

```
from benchmarks.mmlu.mmlu_categories
```